tools/mcp/daemon/sr_od_daemon.py

```
@@ -0,0 +1,228 @@
+    Args:
+        port: 主 server 监听端口,默认 24001。
+
+    Returns:
+        启动结果信息。
+    """
+    existing_proc = find_sr_od_mcp_server_process()
+    if existing_proc:
+        return f"[OK] 主 MCP server 已在运行 (PID: {existing_proc.pid})"
+
+    if is_port_in_use(port):
+        return f"[WARN] 端口 {port} 已被占用，可能有其他程序在使用"
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)